docs/git/cli/lesson.md

```diff
@@ -0,0 +1,81 @@
+---
+id: lesson
+title: Lesson
+sidebar_label: Lesson
+---
+
+import Feedback from "@theme/Feedback";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Learning Objectives
+
+By the end of this class, you should be able to:
+
+- To _clone_ a repository using Git CLI
+- To _commit_, _fork_ a repository, and _push_ changes using Git CLI
+- To view Git's _history log_ using Git CLI
```

Comment:
I don't think I see `log` anywhere in the course material?

(I definitely think we should introduce `log`, but this also means we need to explain navigating the `less` pager, so paging up/down, and `q` to exit - maybe even `/`/`?` for searching... Probably not...)

Also, do we want to introduce `git diff`? If we're introducing navigating `less`, we might as well, I think...

It looks like `git-it` introduces `git diff` in the "Commit To It" step, but it doesn't tell you how to quit out of `git diff`, or how to navigate around it, which is pretty concerning...